azure_functions_worker/bindings/datumdef.py

```diff
@@ -92,6 +94,66 @@ def from_typed_data(cls, td: protos.TypedData):
 
         return cls(val, tt)
 
+    @classmethod
+    def from_rpc_shared_memory(cls, shmem: protos.RpcSharedMemory, shmem_mgr: SharedMemoryManager) -> Optional[Datum]:
+        """
+        Reads the specified shared memory region and converts the read data into a datum object of
+        the corresponding type.
+        """
+        mmap_name = shmem.name
+        offset = shmem.offset
+        count = shmem.count
+        data_type = shmem.type
+        if data_type == protos.RpcDataType.bytes:
+            val = shmem_mgr.get_bytes(mmap_name, offset, count)
+            if val is not None:
+                return cls(val, 'bytes')
+        elif data_type == protos.RpcDataType.string:
```

Comment:
Should we handle stream?

Reply:
There's no stream being exposed here - if you look in `datumdef.py`, those are all the types.
Basically the "stream" is already fully read by the functions host before passing here so it's just bytes now.

Reply:
RpcDataType does have it (there's also one called http which I wasn't aware) is that implicitly mapped to binary in this part of the code?